core/src/mindustry/content/Blocks.java

```diff
@@ -43,9 +43,9 @@
 
     //ores
     oreCopper, oreLead, oreScrap, oreCoal, oreTitanium, oreThorium,
-
+    
```

Comment:
```suggestion
 
```

Reply:
what?

Reply:
you added a few spaces on a line that was empty, if you accept this comment it reverts it back to an empty line 🙂

Reply:
my mistake, it came out by accident 😅